src/Pages/GlobalConfigurations/Authorization/shared/components/K8sObjectPermissions/K8sListItemCard.tsx

```diff
@@ -345,108 +404,148 @@ export default function K8sListItemCard({
             <div className="mb-16">
                 <ReactSelect
                     placeholder="Select cluster"
-                    classNamePrefix="k8s-permission-select-cluster-dropdown"
```

Comment:
Please check this with QA team if they are using it for there testing.

Reply:
Confirmed with Deepak, there is no automation for the auth flow for now